public/pages/Dashboard/containers/Dashboard.js

```diff
@@ -467,7 +467,7 @@ export default class Dashboard extends Component {
       onSelectionChange: this.onSelectionChange,
       selectable: perAlertView
         ? (item) => item.state === ALERT_STATE.ACTIVE
-        : (item) => item.ACTIVE > 0,
+        : (item) => item.ACTIVE >= 0,
```

Comment:
Why are we changing this to `== 0`? I don't think we want to enable selecting the alerts in this case since it will enable certain actions like Acknowledge